src/main/java/uk/gov/hmcts/dts/fact/services/CourtService.java

```diff
@@ -166,6 +167,13 @@ public ServiceAreaWithCourtReferencesWithDistance getNearestCourtsByPostcodeSear
         return new ServiceAreaWithCourtReferencesWithDistance(serviceArea, convert(courts));
     }
 
+    public List<CourtReference> getCourtsByPrefixAndActiveSearch(String prefix, boolean active) {
+        return courtRepository.findCourtBySlugStartingWithAndDisplayed(prefix.toLowerCase(Locale.getDefault()), active)
```

Comment:
Is the returned court list sorted in alphabetical order ready to be used by the front end?